src/cluster/cluster.cc

```diff
@@ -319,7 +319,9 @@ Status Cluster::ImportSlot(Redis::Connection *conn, int slot, int state) {
       conn->SetImporting();
       myself_->importing_slot_ = slot;
       // Set link error callback
-      conn->close_cb_ = std::bind(&SlotImport::StopForLinkError, svr_->slot_import_, conn->GetFD());
+      conn->close_cb_ = [object_ptr = svr_->slot_import_, capture_fd = conn->GetFD()](int fd) {
```

Comment:
Why there is an additional parameter `int fd`?

Reply:
Oh, I notice the type definition. Sorry.

Reply:
`conn->close_cb_ is` a `std::function<void (int)>`, so I add a `int fd`